client/src/index.css

```diff
@@ -25,6 +26,10 @@ time, mark, audio, video {
 	vertical-align: baseline;
 }
 
+h1 {
+	font-family: 'Roboto Slab', serif;
+}
```

Comment:
Let's back up `'Roboto Slab'` with `sans-serif` just to follow the global default backup above :)
It would be nice if we could just do something like `'Roboto Slab', inherit;` but unfortunately css doesn't let us do that with font-familys!